core/adapters/index.ts

```diff
@@ -114,11 +112,15 @@ register("sqldatawarehouse", SQLDataWarehouseAdapter);
 
 export type QueryOrAction = string | dataform.Table | dataform.Operation | dataform.Assertion;
 
+export interface IValidationQuery {
```

Comment:
Why don't we want this in the proto anymore?

Reply:
It's not actually used anywhere that needs to be serialized, it's only used in code, so I've moved it to a typescript interface.